sormas-backend/src/main/java/de/symeda/sormas/backend/event/EventParticipantService.java

```diff
@@ -382,8 +381,8 @@ public boolean isEventParticiapntEditAllowed(EventParticipant eventParticipant)
 			return eventParticipant.getSormasToSormasOriginInfo().isOwnershipHandedOver();
 		}
 
-		return eventParticipantJurisdictionChecker.isInJurisdiction(eventParticipant)
-			&& !sormasToSormasShareInfoService.isEventParticipantOwnershipHandedOver(eventParticipant);
+		// FIXME: Does this intentionally not ask for inJurisdictionOrOwned?
```

Comment:
Seems strange to me. I'd also say it should be isInJurisdictionOrOwned. Maybe @MateStrysewske has an idea why it may be different here.

Reply:
we have now a separation - two methods exposed inJurisdiction and inJurisdictionOrOwned for events and eventParticipants  we use both